build-a-server-rendered-reactjs-application-with-nextjs/README.md

```diff
@@ -0,0 +1,51 @@
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
+[![All Contributors](https://img.shields.io/badge/all_contributors-4-orange.svg?style=flat-square)](#contributors-)
+
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
```

Comment:
Adding the Course Image would be great also adding the course Title as a header as well